example/lib/phyllo_provider.dart

```diff
@@ -73,15 +76,40 @@ class PhylloProvider extends DefaultChangeNotifier {
     _phylloConnect.open();
 
     _phylloConnect.onConnectCallback(
-        onAccountConnected: (account_id, work_platform_id, user_id) {
-      log('onAccountConnected: $account_id, $work_platform_id, $user_id');
-    }, onAccountDisconnected: (account_id, work_platform_id, user_id) {
-      log('onAccountDisconnected: $account_id, $work_platform_id, $user_id');
-    }, onTokenExpired: (user_id) {
-      log('onTokenExpired: $user_id');
-    }, onExit: (reason, user_id) {
-      log('onExit: $reason, $user_id');
-    });
+      onAccountConnected: (account_id, work_platform_id, user_id) {
+        _showToast(
```

Comment:
We can keep toast for testing, once we merge it with the final branch, let's plan to remove this.